INSTALL.md

```diff
@@ -65,18 +67,15 @@ Run the following commands:
 git clone https://github.com/Rangi42/polished-map.git
 cd polished-map
 
-# Build FLTK 1.3.7 with the latest ABI enabled
-# (even if you already have libfltk1.3-dev installed)
-git clone --branch release-1.3.7 --depth 1 https://github.com/fltk/fltk.git
-pushd fltk
-./autogen.sh --prefix="$PWD/.." --with-abiversion=10307
+# Build FLTK 1.4.0
+git clone --branch release-1.4.0rc1 --depth 1 https://github.com/fltk/fltk.git lib/fltk
+pushd lib/fltk
+cmake -D CMAKE_INSTALL_PREFIX="$(realpath "$PWD/../..")" -D CMAKE_BUILD_TYPE=Release -D FLTK_GRAPHICS_CAIRO=1 -D FLTK_BACKEND_WAYLAND=0
```

Comment:
Updated this to clone into `lib/fltk` instead of `fltk` (to match the Windows instructions).
(also updated in .github/workflows/main.yml so that the `cmake` command is identical)